nixos/tests/nixos-test-driver/multi-os.nix

```diff
@@ -0,0 +1,79 @@
+{ lib, hostPkgs, ... }:
+
+let
+
+  nixThePlanet = builtins.getFlake "github:MatthewCroughan/NixThePlanet/c9d159dc2e0049e7b250a767a3a01def52c3412b";
```

Comment:
I don’t think we should use `builtins.getFlake` in Nixpkgs, especially to a repository that contains a bunch of legally dubious stuff. (But I assume this is just a proof‐of‐concept and the intention was to pull the relevant functionality in‐tree.)